hack/test-end-to-end.sh

```diff
@@ -168,25 +168,38 @@ echo "[INFO] Using images:              ${USE_IMAGES}"
 
 # Start All-in-one server and wait for health
 # Specify the scheme and port for the listen address, but let the IP auto-discover.	Set --public-master to localhost, for a stable link to the console.
+echo "[INFO] Minting certificates for the OpenShift server"
+LIKELY_DEFAULT_SERVER_IP=`ifconfig | grep -Ev "(127.0.0.1|172.17.42.1)" | grep "inet " | head -n 1 | awk '{print $2}'`
```

Comment:
should we do this computation, then set API_HOST to it as the default instead of "localhost", and pass in `--master="${API_SCHEME}://${API_HOST}:${API_PORT}"`?